src/Billing/Enum/ConstantEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Billing\Enum;
+
+use ArrowSphere\PublicApiClient\AbstractEnum;
+
+abstract class ConstantEnum extends AbstractEnum
```

Comment:
if this class is abstract, please name it AbstractConstantEnum
See https://www.php-fig.org/bylaws/psr-naming-conventions/